Plots/Contours/NCL_minmax_3.py

```diff
@@ -0,0 +1,291 @@
+"""
+NCL_minmax_3.py
+===============
+This script illustrates the following concepts:
+   - Calculating the local minima/maxima of your data
+   - Adding text strings at local minima/maxima locations
+   - Generating dummy data using "generate_2d_array"
+   - Setting the background color for a text box
+   - Turning on the perimeter of a text box
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/minmax_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/minmax_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+from geocat.viz import cmaps as gvcmaps
+
+################################################################
+# Definition of generate_2d_array and helper functions from https://github.com/NCAR/pyngl/blob/develop/src/ngl/__init__.py
+
+#  Globals for random number generator for generat_2d_array
+dfran_iseq = 0
+dfran_rseq = [.749, .973, .666, .804, .081, .483, .919, .903,   \
```

Comment:
the '\' in this block can be taken out along with the trailing whitespace that was added in pyngl

Reply:
Updated. Lmk if this is what you had in mind!